RecoVertex/PrimaryVertexProducer/interface/PrimaryVertexProducer.h

```diff
@@ -82,6 +82,9 @@ class PrimaryVertexProducer : public edm::stream::EDProducer<> {
   edm::ParameterSet theConfig;
   bool fVerbose;
 
+  bool isRecoveryIteration;
```

Comment:
```suggestion
  bool fRecoveryIteration;
```

to follow the style of naming of other flags in this module